src/Groups/cosets.jl

```diff
@@ -68,6 +83,21 @@ Return the coset `gH`.
 !!! note
     Since GAP supports right cosets only, the underlying GAP object of
     `left_coset(H,g)` is the right coset `H^(g^-1) * g`.
+
+# Examples
+```jldoctest
+julia> G=symmetric_group(5)
+Sym( [ 1 .. 5 ] )
+
+julia> g = perm(G,[3,4,1,5,2])
+(1,3)(2,4,5)
```

Comment:
Maybe simplify here and elsewhere to this?
```suggestion
julia> g = perm([3,4,1,5,2])
(1,3)(2,4,5)
```
Or you could reduce the whole example to a single line, e.g.
```
gH = left_coset(symmetric_group(3), @perm (1,3)(2,4,5))
```